script/seed.js

```diff
@@ -478,7 +474,7 @@ async function seed() {
 
   for (let i = 0; i < 995; i++) {
     productArray.push(
-      Product.create({
+      await Product.create({
```

Comment:
we actually don't want an await here because the product array should be an array of promises instead of resolved  promises so that we can use Promise.all on that array and it can resolve all the promises simultaneously instead of one at a time in this loop